### DISCUSSION TITLE
What is the Budibase equivalent to nuBuilder lookup?

### DISCUSSION THREAD

**Discussion by m2acgi** [Budibase/budibase#6772] [Help] [answered]
Click the paly button in the upper right corner of the gif to play.

![2022-07-16_11-29-02](https://user-images.githubusercontent.com/34497122/179337479-0f7d768e-d37e-423d-a563-043540bda22b.gif)

**Comment by melohagan** [comment, marked as answer]
Hi @m2acgi 

There is no specific 'lookup' component. You should be able to build something similar with the following though:

- Modal (https://docs.budibase.com/docs/screens#modal-screen)
- Table (https://docs.budibase.com/docs/table)

**Reply by m2acgi** [reply]
Thanks!
